<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="proccess">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="volume">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>390</y>
      <width>22</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>79</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="openBtn">
       <property name="text">
        <string>open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playBtn">
       <property name="text">
        <string>play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseBtn">
       <property name="text">
        <string>pause</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>691</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
